chain/Dockerfile

```diff
@@ -10,4 +10,16 @@ COPY chain/docker-config/run.sh .
 
 RUN make install && make faucet
 
+FROM debian:buster
+
+ENV WORKDIR=/chain
+WORKDIR ${WORKDIR}
+
+COPY --from=builder /go-owasm /go-owasm
+COPY --from=builder /chain/docker-config ${WORKDIR}/docker-config
+COPY --from=builder /chain/scripts ${WORKDIR}/scripts
```

Comment:
I think we don't need this folder to the image.

Reply:
Just to be sure, are you talking about `/chain/scripts` directory ?

Reply:
Yes, it's just a folder that contains convenience scripts for development.

Reply:
Got it, it's removed now